core-bundle/src/Command/SymlinksCommand.php

```diff
@@ -35,6 +35,10 @@
  */
 class SymlinksCommand extends Command
 {
+    public const PACKAGES = [
+        'scrivo/highlight.php' => 'vendor/scrivo/highlight.php/styles',
```

Comment:
Was the rename from `highlight_php` to `highlight.php` on purpose?

Reply:
But why did we have an underscore in the name then? Did the dot cause any problems?

Reply:
Yes, but we currently symlink it to `public/vendor/scrivo/highlight_php/styles` and with this change we would symlink it to `public/vendor/scrivo/highlight.php/styles` instead.

Reply:
Good catch. I have to investigate.

Reply:
I have fixed the symlink in 4087dc79a8e6a534a3aaed19a68436f9cbc03558. Also, I have removed the `SymlinksCommand::PACKAGES` constant for now because finding a way to symlink vendor assets is a different topic that might or might not be solved with a class constant. We should discuss it separately once there is more than one package.